RecoMuon/MuonRechitClusterProducer/python/cscRechitCluster_cfi.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+ca4cscRechitClusters = cms.EDProducer(
+        "CSCRechitClusterProducer",
+        rParam = cms.double(0.4),
+        nRechitMin = cms.int32(50)
+)
```

Comment:
please import the auto-generated `cscRechitClusterProducer_cfi` and clone from it
```suggestion
import  RecoMuon.MuonRechitClusterProducer.cscRechitClusterProducer_cfi as _mod
ca4cscRechitClusters = _mod.cscRechitClusterProducer.clone()
```

Reply:
comments in this file are still not addressed

Reply:
I'm fine if the file cscRechitCluster_cfi.py  is removed.
Actually, now I realize that it should also be renamed to cscRechitCluster_cff.py (cfi is supposed to define just one module instance, in most cases)

Reply:
@slava77 
It's probably easier to keep the python file in case we need to change the values. 
How about we change the name to `RechitCluster_cff.py` with the following content?
```
import RecoMuon.MuonRechitClusterProducer.CSCrechitClusterProducer_cfi as CSCcluster
import RecoMuon.MuonRechitClusterProducer.DTrechitClusterProducer_cfi as DTcluster

ca4CSCrechitClusters= CSCcluster.CSCrechitClusterProducer.clone(
    recHitLabel = "csc2DRecHits",
    nRechitMin  = 50,
    rParam      = 0.4,
    nStationThres = 10,
)
ca4DTrechitClusters = DTcluster.DTrechitClusterProducer.clone(
    recHitLabel = "dt1DRecHits",
    nRechitMin  = 50,
    rParam      = 0.4,
    nStationThres = 10,
)
```
And I will update the names in these two lines for easier reading of the producer names, and keeping the producer name capitalized:
```
DTrechitClusterProducer
CSCrechitClusterProducer
```
https://github.com/cms-sw/cmssw/blob/420d500cd7c08d6d682133bf8ae847c847d1b140/RecoMuon/MuonRechitClusterProducer/plugins/RechitClusterProducer.cc#L264-L265

Reply:
>  How about we change the name to RechitCluster_cff.py with the following content?

`rechitCluster_cff.py` should be OK (note the lower case)